.github/dependabot.yml

```diff
@@ -0,0 +1,21 @@
+version: 2
+updates:
+  - package-ecosystem: npm
+    directories: 
+      - "/"
+      - "/ember-liveagent"
+      - "/test-app"
+    schedule:
+      interval: daily
+      time: '04:00'
+    open-pull-requests-limit: 10
+    versioning-strategy: increase
```

Comment:
The [doc](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#versioning-strategy) says that increase should be used for apps, and widen for libraries. Since the monorepo has both app and library, then the default ✂️ should be used instead? 🤔